samcli/commands/deploy/guided_context.py

```diff
@@ -203,3 +212,17 @@ def run(self):
                 confirm_changeset=self.confirm_changeset,
                 capabilities=self._capabilities,
             )
+
+    def _get_parameter_default_value(self, parameter_key, parameter_properties, parameter_override_from_cmdline):
```

Comment:
we should just call this value, since we are fetching from config file if present, not just default values.

Reply:
Changed the function name here.